lib/HTTP/Tiny.pm

```diff
@@ -115,7 +117,7 @@ sub new {
         max_redirect => 5,
         timeout      => defined $args{timeout} ? $args{timeout} : 60,
         keep_alive   => 1,
-        verify_SSL   => $args{verify_SSL} || $args{verify_ssl} || 0, # no verification by default
+        verify_SSL   => $args{verify_SSL} // $args{verify_ssl} // _verify_SSL_default(),
```

Comment:
Use of `//` bumps the minimum required Perl, which is currently v5.6.  I have no opinion about this change, but if we're going to make this change, this file should have `use v5.10.1` in the opening and `[MinimumPerl]` should be added to the `dist.ini` file.  Possibly `[OnlyCorePrereqs]` should be added as well with appropriate options to ensure HTTP::Tiny can continue to be dual-lifed.

Reply:
Fix to avoid `//`